istio/istio-install/base/kustomization.yaml

```diff
@@ -1,5 +1,141 @@
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
+# Each entry in this list results in the creation of
+# one ConfigMap resource (it's a generator of n maps).
+configMapGenerator:
```

Comment:
cool use of configMapGenerator that includes a number of resources

Reply:
The [helm-kustomize convertor](https://github.com/ContainerSolutions/helm-convert) introduced this to me.